core/meta/inc/TClass.h

```diff
@@ -309,12 +311,14 @@ friend class TProtoClass;
    TClass(const TClass& tc) = delete;
    TClass& operator=(const TClass&) = delete;
 
+   void LoadClassInfo() const;
```

Comment:
Why is moving LoadClassInfo from one part of the private section to another useful? (or am I missing something?)